scripts/dup_audit.py

```
@@ -0,0 +1,530 @@
+    total_loc = sum(fi.non_empty_loc for fi in file_infos)
+
+    exact = compute_exact_duplicates(file_infos)
+    near = compute_near_duplicates(file_infos)
+    blocks = compute_block_clones(file_infos)
+    func_eq = compute_functional_equivalents(file_infos)
+    env_forks = compute_env_forks(file_infos)
+
+    # Estimate LOC savings
+    saved_loc = 0
+    # exact duplicates: for each cluster with k copies, save (k-1)*avg(loc)
+    for h, group in exact.items():
```

Ruff (0.12.2)

Loop control variable `h` not used within loop body

Rename unused `h` to `_h`

(B007)